Turner.Infrastructure.Mediator/Error.cs

```diff
@@ -0,0 +1,9 @@
+namespace Turner.Infrastructure.Mediator
+{
+    public class Error
+    {
+        public string PropertyName { get; set; }
```

Comment:
I have always been up in the air about this. I guess it is assumed that you leave the PropertyName blank if it cant be tied to a single property?

Reply:
@justinobney Yeah, that's been how we've always done it...